scripts/tcpperf_plot.py

```diff
@@ -1,9 +1,11 @@
-from collections import namedtuple
 import csv
+from collections import namedtuple
```

Comment:
I would prefer if these stayed in alphabetical order.

Reply:
pylint or pycodestyle was complaining about this in python 3.9 if I'm not mistaken...